tests/reg_issue2365/reg_issue2365.fz.expected_err

```diff
@@ -3,7 +3,7 @@
   for _ := for j in 0.. else _ := j
 ----------------------------------^
 Feature not found: 'j' (no arguments)
-Target feature: 'reg_issue2365.loop.else'
+Target feature: 'reg_issue2365.else'
```

Comment:
Why is the output different here? I think it should still be `loop.else`?

Reply:
I was a bit surprised here as well. The code for the `else` part is duplicated, so we have different `else`'s depending on whether the loop body is executed zero times or at least once. My assumption is that this error is now found first in the version when the loop body is never executed, but I did not verify that my assumption is correct.

Reply:
@fridis sounds plausible